kura/test/org.eclipse.kura.core.configuration.test/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,24 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: org.eclipse.kura.core.configuration.test
+Bundle-SymbolicName: org.eclipse.kura.core.configuration.test;singleton:=true
+Bundle-Version: 2.1.0.qualifier
+Bundle-Vendor: Eclipse Kura
+Bundle-RequiredExecutionEnvironment: JavaSE-1.6
+Import-Package: org.eclipse.kura;version="1.2.1",
+ org.eclipse.kura.configuration;version="1.1.0",
```

Comment:
@markoer I'm confused if we really need a MANIFEST at this stage (see comment below for the pom file).
However, please take care of importing the org.eclipse.kura, org.eclipse.kura.configuration, org.eclipse.kura.crypto, org.eclipse.kura.system with proper ranges as described in the OSGi Semantic Versioning paper.

The  org.eclipse.kura.core.* packages are internal packages and we currently do not update their version properly so you should not rely on their version.
Do you really need all of them in the MANIFEST?